data/lui2020viral.yaml

```diff
@@ -0,0 +1,184 @@
+title: Viral dynamics of SARS-CoV-2 across a spectrum of disease severity in COVID-19
+doi: 10.1016/j.jinf.2020.04.014
+description: >
+  Lui et al. report on a prospective cohort study of patients with variable
+  disease severity. Viral loads for positive samples were extracted from Tbl. 1 in
+  the supplementary material, and negatives were extracted manually from Fig. 1 and
+  a figure in the supplementary material. The level of quantification was extracted
+  from methods in the supplementary material.
+
+  Data for other specimen types, including nasopharyngeal swabs, sputum, plasma, and
+  urine, are also available in the source but have not yet been included here.
+analytes:
+  2019-nCoV_N1:
+    description: Gene copies of the N1 gene quantified using RT-qPCR in stool samples.
+    limit_of_detection: unknown
+    limit_of_quantification: 694.1199999839523
+    specimen: stool
+    gene_target: N1
+    unit: gc/mL
+    reference_event: symptom onset
+    biomarker: SARS-CoV-2
+participants:
+- attributes:
+    severity: critical
```

Comment:
`severity` is currently not in the schema.

Reply:
Do we want to standardize all the attributes in the schema? E.g., we don't have `study_arm` for `natarajan2022` or `vaccinated` for `Liu2024Longitudinal`?

Reply:
I see. Actually, I should add `vaccinated` instead of `inpatient` to the schema. We don't have to include the information is not commonly shared by studies.

Reply:
Till, could you help add `vaccinated` to the schema and remove the `inpatient` in this pull request? So I do not have to open another pull request?

Reply:
Sure thing.